### ISSUE TITLE
Getting error while train of lengths array has to be in decreasing order

### ISSUE THREAD

**Issue by Garima-Mishra** [OpenNMT/OpenNMT-py#215]
Models.py", line 257, in forward
    packed_emb = pack(emb, lengths)
  File "/usr/local/lib/python2.7/dist-packages/torch/nn/utils/rnn.py", line 78, in pack_padded_sequence
    raise ValueError("lengths array has to be sorted in decreasing order")
ValueError: lengths array has to be sorted in decreasing order

**Comment by srush** [comment]
Yeah, this is a torchtext change. See this PR for a solution https://github.com/OpenNMT/OpenNMT-py/pull/200
